### PR TITLE
add FAQ about rust-analyser option for completion

### LINKED ISSUE #10: Mention loadOutDirsFromCheck in FAQ for rust-analyzer completion
Some users seem to have trouble with auto-completion. Setting `"rust-analyzer.cargo.loadOutDirsFromCheck": true` might resolve the problem as mentioned in many rust-analyzer issues.

### PR DESCRIPTION
**Pull request by extrawurst**
closes #10

### CODE REVIEW

File: src/faq.md [hunk 1]
```diff
@@ -115,3 +115,17 @@ impl AnotherNativeScript {
 }
 
 ```
+
+## Intellisense with rust-analyzer
+
+**Question**
+
+`godot-rust` generates most of the gdnative type's code at compile-time. IDE's using [rust-analyzer](https://github.com/rust-analyzer/rust-analyzer) struggle to autocomplete those types:
```
Comment (ghost) [inline src/faq.md]: Typo: `IDEs`?

File: src/faq.md [hunk 2]
```diff
@@ -115,3 +115,17 @@ impl AnotherNativeScript {
 }
 
 ```
+
+## Intellisense with rust-analyzer
```
Comment (ghost) [inline src/faq.md]: I believe Intellisense is a Microsoft-specific technology and might be a trademark. I'd prefer "Auto-completion" here.

Comment (extrawurst) [reply src/faq.md]: fixed

File: src/faq.md (hunk 1 above)
Comment (extrawurst) [reply src/faq.md]: fixed

**Review by ghost — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**